src/components/Artists/ArtistsList/ArtistData.js

```diff
@@ -0,0 +1,62 @@
+export const ArtistData = [
```

Comment:
this should not be here - move this to main component and pass with props from there

Reply:
Fixed